src/main/java/com/github/ksprojects/ZkCopy.java

```diff
@@ -11,11 +11,13 @@
     private static final Logger LOGGER = Logger.getLogger(ZkCopy.class);
     private static final int DEFAULT_THREADS_NUMBER = 10;
     private static final boolean DEFAULT_REMOVE_DEPRECATED_NODES = true;
+    private static final boolean DEFAULT_IGNORE_EPHEMERAL_NODES = false;
```

Comment:
I think that ephemeral nodes should not be copied by default. That's rarely is expected behavior. Let's invert settings: use `copyEphemeralNodes` flag.